src/gui/SimMenuBar.java

```diff
@@ -223,13 +235,17 @@ private void toggleUnderlayImage() {
 			String imgFile = null;
```

Comment:
`enableBgImage` will be `null` if the settings don't have `GUI.UnderlayImage.fileName = foo`, which will crash on startup.